cmd/skunk/adapter/in/networkAdapter/networkAdapter.go

```diff
@@ -0,0 +1,250 @@
+package networkAdapter
+
+import (
+	"encoding/json"
+	"fmt"
+	"sync"
+	"time"
+
+	cretztor "github.com/cretz/bine/tor"
+
+	"github.com/scherzma/Skunk/cmd/skunk/adapter/in/peer"
+	"github.com/scherzma/Skunk/cmd/skunk/adapter/in/tor"
+	"github.com/scherzma/Skunk/cmd/skunk/application/port/network"
+	"github.com/scherzma/Skunk/cmd/skunk/util"
+)
+
+// Constants for configuration of Tor
+const (
+	SocksPort            = "9055"
+	LocalPort            = "1111"
+	RemotePort           = "2222"
+	ReusePrivateKey      = true // reuse private key for constant onion address
+	DeleteDataDirOnClose = false
+	UseEmbedded          = true // use embedded tor process
+)
+
+var (
+	networkAdapter *NetworkAdapter
+	once           sync.Once
+)
+
+// NetworkAdapter connects the main logic to the tor peer network
+type NetworkAdapter struct {
```

Comment:
Should that be "networkAdapter"? To be private.

Reply:
Why should that be private?